subsys/shell/shell_rtt.c

```diff
@@ -17,6 +21,8 @@ SHELL_DEFINE(shell_rtt, "rtt:~$ ", &shell_transport_rtt,
 
 LOG_MODULE_REGISTER(shell_rtt, CONFIG_SHELL_RTT_LOG_LEVEL);
 
+static bool rtt_blocking;
```

Comment:
shouldn't this be `volatile` ?

Reply:
i don't think it's needed. It's read only once in `write` function.